kokoro/presubmit.sh

```diff
@@ -57,8 +57,10 @@ function run_unit_tests() {
 }
 
 function run_integration_tests(){
-    echo Running integration tests.
+    echo Running basic integration test.
     python3 -m pytest dataflux_pytorch/integration_tests/integration_test.py -vv --junit-xml="${KOKORO_ARTIFACTS_DIR}/integration_tests/sponge_log.xml" --log-cli-level=DEBUG
+    echo Running checkpoint integration test.
+    python3 dataflux_pytorch/benchmark/lightning_checkpoint_benchmark.py --enable-multipart --project=dataflux-project --ckpt-dir-path=gs://jd-upload-west/ --dataflux-ckpt --layers=10 --steps=5
```

Comment:
We may want to use a dedicated bucket for this, with a retention rule enabled that automatically deletes files after a while, so that old checkpoint files don't pile up and take up space

Reply:
my bucket does have those rules but i'll make a more officially named one for this